blueprints/add_drink.py

```diff
@@ -0,0 +1,87 @@
+from datetime import datetime
+
+from flask import Blueprint, session, render_template
+from bson import ObjectId
+
+from .utils import *
+from .api import get_ingredients_by_type, get_flavors
+
+add_drink = Blueprint('add_drink', __name__)
+
+
+@add_drink.route('/add_cocktail/', methods=["GET", "POST"])
```

Comment:
Remember what we discussed about using restful routes. I'll expect this route to be `/cocktails/new`. You don't handle POST request here so methods should only include `GET`